adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctionsSuite.scala

```diff
@@ -122,12 +123,18 @@ class AlignmentRecordRDDFunctionsSuite extends ADAMFunSuite {
 
   sparkTest("round trip from ADAM to SAM and back to ADAM produces equivalent Read values") {
     val reads12Path = Thread.currentThread().getContextClassLoader.getResource("reads12.sam").getFile
-    val rdd12A: RDD[AlignmentRecord] = sc.loadAlignments(reads12Path)
+    val ardd = sc.loadBam(reads12Path)
+    val rdd12A = ardd.rdd
+    val sd = ardd.sequences
+    val rgd = ardd.recordGroups
 
     val tempFile = Files.createTempDirectory("reads12")
-    rdd12A.adamSAMSave(tempFile.toAbsolutePath.toString + "/reads12.sam", asSam = true)
+    rdd12A.adamSAMSave(tempFile.toAbsolutePath.toString + "/reads12.sam",
+      sd,
+      rgd,
+      asSam = true)
 
-    val rdd12B: RDD[AlignmentRecord] = sc.loadBam(tempFile.toAbsolutePath.toString + "/reads12.sam/part-r-00000")
+    val rdd12B = sc.loadBam(tempFile.toAbsolutePath.toString + "/reads12.sam/part-r-00000").rdd
```

Comment:
Both of these still work for me
```scala
val rdd12B = sc.loadBam(tempFile.toAbsolutePath.toString + "/reads12.sam/part-r-00000")                                                                                        
val rdd12B: RDD[AlignmentRecord] = sc.loadBam(tempFile.toAbsolutePath.toString + "/reads12.sam/part-r-00000") 
```
I assume this is the implicit at work, and the ```.rdd``` isn't actually necessary here?